packages/eslint-plugin-turbo/lib/rules/no-undeclared-env-vars.ts

```diff
@@ -1,10 +1,20 @@
 import path from "node:path";
+import { readFileSync } from "node:fs";
 import type { Rule } from "eslint";
 import type { Node, MemberExpression } from "estree";
-import { logger } from "@turbo/utils";
+import { type PackageJson, logger, searchUp } from "@turbo/utils";
+import { frameworks } from "@turbo/types";
 import { RULES } from "../constants";
 import { Project, getWorkspaceFromFilePath } from "../utils/calculate-inputs";
 
+/** set this to true if debugging this rule */
+const debugging = "".length === 0;
```

Comment:
This is breaking some tests that expect valid JSON on stdout when running eslint with `--json`. 

Leaving this in is fine - but let's change this to read from `process.env.RUNNER_DEBUG`. We use this in our jest configs ([example](https://github.com/vercel/turborepo/blob/main/packages/eslint-plugin-turbo/jest.config.js#L12-L13)) and you can set it locally to debug, and also if you debug a github action run it will output debug logs there as well.

Reply:
this is a mistake - it should be off when merged and I just forgot.  thank you! fixed in 0fbcc9580071f3a31bc194aa57cc772d04f5a951